players/mpris2/mpris2.py

```diff
@@ -58,8 +58,10 @@ def _get_player_from_bus_names(self, names):
         Arguments:
         - `names`: list of bus names
         """
-        return [player_info_from_name(name[len(MPRIS2_PREFIX):]) for name in names
-                if name.startswith(MPRIS2_PREFIX)]
+        return [player_info_from_name(name[len(MPRIS2_PREFIX):])
+                for name in names if name.startswith(MPRIS2_PREFIX) and
+                # skip self: APP_MPRIS2_NAME (in daemon/main.py)
```

Comment:
Couldn't we move that constant to `python/consts.py`?  This way we could filter out `APP_MPRIS2_NAME` explicitly.

Actually there are three constants in `daemon/main.py` that would probably be better found in `python/consts.py`, that could be done in this commit or a separate one.